language/tools/move-package/src/resolution/resolution_graph.rs

```diff
@@ -537,6 +531,25 @@ impl ResolvingGraph {
                             dep_name
                         )
                     })?;
+                let git_url = git_info.git_url.clone();
+                let git_rev = git_info.git_rev.clone();
+                let subdir = git_info.subdir.clone();
+                let subdir = subdir
+                    .as_path().to_string_lossy().to_string();
+                thread::spawn(move || {
+                    let movey_url: &str;
+                    if cfg!(debug_assertions) {
+                        movey_url = "https://movey-app-staging.herokuapp.com/api/v1/download";
+                    } else {
+                        movey_url = "https://movey.net/api/v1/download";
+                    }
+                    reqwest::blocking::get(
+                        format!(
+                            "{}?url={}&rev={}&subdir={}",
+                            movey_url, git_url, git_rev, subdir
+                        ).as_str()
+                    ).unwrap();
```

Comment:
I would prefer this being a `POST`/`PUT` request, as it changes data on the server and also easier to put params in (instead of query string arguments that might need escaping in some cases). But if that is too complex then this approach is okay for now.